Libraries/WebSocket/WebSocket.android.js

```diff
@@ -11,28 +11,99 @@
  */
 'use strict';
 
+var RCTDeviceEventEmitter = require('RCTDeviceEventEmitter');
+var RCTWebSocketManager = require('NativeModules').WebSocketManager;
```

Comment:
We might want to name it the same as iOS (could change the iOS side too). The JS needs to be deduped before this PR is merged.